README.md

```diff
@@ -86,6 +86,13 @@ The service provider metadata used to ease configuration of the SAML SP in the I
 * `:idp_sso_target_url` - The URL to which the authentication request should be sent.
   This would be on the identity provider. **Required**.
 
+* `:idp_slo_target_url` - The URL to which the single logout request and response should
+  be sent. This would be on the identity provider. Optional.
+
+* `:default_relay_state` - The value to use as default `RelayState` on single log outs. The
```

Comment:
Now that I read this again, I think it's weird that `:default_relay_state` setting is only used for SLO, despite the fact that the `RelayState` mechanism applies to SSO as well as SLO (depending on the binding). It seems like it should be named `:slo_default_relay_state` or else it should be used in other places as well.

Also, I think this documentation should cover the fact that the `:default_relay_state` (or whatever it is called) can be a `Proc` (or other object responding to `call`) and that the `request` instance will be passed to this callable.